services/tutorials/ethereum/authenticate-with-jwt.md

```diff
@@ -0,0 +1,372 @@
+---
+description: Apply a JWT to authenticate an API request.
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Authenticate with JWT
+
+This tutorial demonstrates how to create and apply a JSON Web Token (JWT) to authenticate an 
+[`eth_blockNumber`](../../api/networks/ethereum/json-rpc-methods/eth_blocknumber.mdx) API request 
+with Node.js. 
+
+Developers can configure the expiry time and scope of JWTs to enhance the security profile of their dapps.
+
+## Prerequisites
+
+- [Node](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 20+
+- A text editor (for example, [VS Code](https://code.visualstudio.com/))
+- An [Infura account](https://app.infura.io/register)
+
+## Steps
+
+### 1. Initiate your project
+
+Create a new directory for your project:
+
+```bash
+mkdir infura-jwt-demo
+```
+
+Next, change into the `infura-jwt-demo` directory:
+
+```bash
+cd infura-jwt-demo
+```
+
+Initialize a new Node.js project:
+
+```bash
+npm init -y
+```
+
+Install the required dependencies:
+
+```bash
+npm install axios jsonwebtoken dotenv
+```
+
+### 2. Create a key pair 
+
+#### 2.1. Generate your private key
+
+Generate your private key using the RSA (PKCS #8) or ES256 algorithm:
+
+<Tabs>
+	<TabItem value="RSA" default>
+
+```bash
+openssl genpkey                 \
+  -algorithm RSA                \
+  -out private_key.pem          \
+  -pkeyopt rsa_keygen_bits:2048 \
+  -outform PEM
+ ```
+
+  </TabItem>
+  <TabItem value="ES256">
+
+```bash
+openssl ecparam    \
+  -genkey          \
+  -name prime256v1 \
+  -noout           \
+  -out private_key.pem
+```
+
+  </TabItem>
+</Tabs>
+
+#### 2.2. Generate your public key
+
+Generate your public key from your RSA or ES256 private key:
+
+<Tabs>
+	<TabItem value="RSA" default>
+
+```bash
+openssl rsa           \
+  -in private_key.pem \
+  -pubout             \
+  -out public_key.pem
+```
+
+  </TabItem>
+  <TabItem value="ES256">
+
+```bash
+openssl ec            \
+  -in private_key.pem \
+  -pubout             \
+  -out public_key.pem
+```
+
+  </TabItem>
+</Tabs>
+
+### 3. Set up your `.env` file
+
+#### 3.1. Update the Infura dashboard
+
+In the [Infura dashboard](https://app.infura.io/), under **API Keys**, select the key you want to use for authentication. Go to its **Settings** tab. Under **Requirements**, fill out these fields:
+
+- **JWT PUBLIC KEY NAME** - Provide a unique name for your JWT public key, which can help you manage multiple keys.
+- **JWT PUBLIC KEY** - Paste the entire contents of the `public_key.pem` file.
+
+:::note
```

Comment:
Maybe add a space after this and after block of text (for consistency with the other admonitions)